eledmac.dtx

```diff
@@ -1772,8 +1772,8 @@
 % The default series order is the one called with the \verb+series+ option of the package, or, if this option is not used, A, B, C, D, E, Z. 
 % Series order determines footnotes order.
 %
-% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order inside the document.
-% You can use \cs{seriesatbegin}\marg{<s>} to push the series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push the series \verb+<s>+ at the end.
+% \DescribeMacro{seriesatbegin}\DescribeMacro{seriesatend} However in some specific cases, you need to change the series order at some point inside the document.
+% You can use \cs{seriesatbegin}\marg{<s>} to pull a given series \verb+<s>+ at the beginning, and \cs{seriesatend}\marg{<s>}  to push it \verb+<s>+ at the end.
```

Comment:
>  En fait, on prend une s�rie appel�e <s>, et on peut d�cider de lui appliquer une commande qui la remonte au d�but ou une autre qui la descend � la fin.

c'est tout � fait cela

> J'ai un peu chang� de cette fa�on: ```` % You can use \cs{seriesatbegin}\marg{<s>} to pull up a given series % \verb+<s>+ at the beginning, or \cs{seriesatend}\marg{<s>} to push % it down \verb+<s>+ at the end. ```` 

c'est l� o� je ne comprend pas la phrase. La premi�re parti est ok. Le COD de pull up a  "a given series `<s>`". Par contre dans la deuxi�me partie le COD de "push down" est � la fois "it" et "\verb+<s>+". Je suis donc d'avis de supprimer le dernier "\verb+<s>+".